### DISCUSSION TITLE
建议在程序运行时的提示，写一个赞助商，以表感谢

### DISCUSSION THREAD

**Discussion by chika0801** [nxtrace/NTrace-core#156] [General]
![无标题](https://github.com/nxtrace/Ntrace-core/assets/88967758/c8c2a7d8-1dea-4813-a0e5-43aac8e7e0f9)

开头位置可以提下赞助商名称，感觉比只在README中写能让更多人看下，也不会有太大影响。

如果赞助商自己说不需要，那也无妨了。

**Comment by tsosunchia** [comment]
群友chika0801你好，
我们已经在考虑你的建议了，在设想一个不影响美观，又能显示SPONSOR的两全其美方法，
目前的后端和前端，还需要做较大的调整才能显示他（因为大部分情况下，我们的用户是通过CF CDN访问的），
感谢你的建议。
后续有进展时将在此讨论区跟进。

**Comment by tsosunchia** [comment]
https://github.com/nxtrace/NTrace-V1/commit/dbe42d669c916af9e536ab345e37e8f6a2f265d9

已实现
